src/index.html

```diff
@@ -2,16 +2,170 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldsetstyle">
+        <legend>Personal information</legend>
+        <label class="form-field" for="surname">
+          Surname:
+          <input
+          id="surname"
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
```

Comment:
```suggestion
          <input
            id="surname"
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```
![image](https://user-images.githubusercontent.com/96409563/217753415-a806c7fc-0ed1-4ea0-8449-94b9bcc2f5f2.png)